code/modules/clothing/factions/hardliners.dm

```diff
@@ -0,0 +1,181 @@
+	blood_overlay_type = "armor"
+
+/obj/item/clothing/suit/armor/hardliners/jacket
+	name = "hardliners armored kutte"
+	desc = "A leather Kutte with a slim Type I armored vest, painted in a classic white associated with the Hardliners. The patch of the Hardliner movement can be seen behind the leather kutte, a nostalgic callback to the leather outfits used by the civilians of Gorlex VII."
+	icon_state = "hl_jacket"
+	item_state = "armor"
+	icon = 'icons/obj/clothing/faction/hardliners/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/suits.dmi'
+	blood_overlay_type = "armor"
+
+/obj/item/clothing/suit/armor/hardliners/sergeant
+	name = "hardliners sergeant jacket"
+	desc = "An armored jacket typically worn by sergeant of the Hardliners. They're reminiscent of the garb worn by old Gorlex navymen, prior to its destruction."
+	body_parts_covered = CHEST|GROIN|ARMS
+	icon_state = "hl_sergeant"
+	item_state = "hl_sergeant"
+	blood_overlay_type = "coat"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+
+/obj/item/clothing/suit/toggle/armor/vest/hardliners
+	name = "hardliners captain coat"
+	desc = "An imposing armored coat worn by captains of Hardliner fleets, hand-designed by Cybersun tailors to provide maximum protection to its wearer."
+	body_parts_covered = CHEST|GROIN|ARMS
+	icon_state = "hl_captain"
+	item_state = "hl_captain"
+	icon = 'icons/obj/clothing/faction/hardliners/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/suits.dmi'
+	blood_overlay_type = "coat"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	togglename = "buttons"
+
+///////////////
+//Spacesuits//
+//////////////
+
+/obj/item/clothing/head/helmet/space/hardsuit/syndi/hl
+	name = "white-red hardsuit helmet"
+	desc = "An advanced dual-mode helmet derived from ICW-era advanced special operations helmets, its red partly replaced by white. It is in EVA mode. Manufactured by Cybersun."
+	alt_desc = "An advanced dual-mode helmet derived from ICW-era advanced special operations helmets, its red partly replaced by white. It is in combat mode. Manufactured by Cybersun."
+	icon_state = "hardsuit1-hl"
+	item_state = "hardsuit1-hl"
+	icon = 'icons/obj/clothing/faction/hardliners/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/head.dmi'
+	hardsuit_type = "hl"
+
+/obj/item/clothing/suit/space/hardsuit/syndi/hl
+	name = "white-red hardsuit"
+	desc = "An advanced dual-mode hardsuit derived from ICW-era advanced special operations hardsuits, its red partly replaced by white. It is in EVA mode. Manufactured by Cybersun."
+	alt_desc = "An advanced dual-mode hardsuit derived from ICW-era advanced special operations hardsuits, its red partly replaced by white. It is in combat mode. Manufactured by the Cybersun."
```

Comment:
Slight grammar error at the end there ("Manufactured by the Cybersun"). I'd actually put the full name on both modes ("Manufactured by Cybersun Biodynamics.")